courses/machine_learning/deepdive2/supplemental/solutions/object_detection_with_automl.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<p>Remove this title (the description of the notebook below can go directly after the learning objectives).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>done</p>